src/Microsoft.ML.CpuMath/AvxIntrinsics.cs

```diff
@@ -333,34 +418,31 @@ public static unsafe void MatMulX(float[] mat, float[] src, float[] dst, int cro
                         int col1 = *ppos;
                         int col2 = col1 + 4 * ccol;
                         Vector256<float> x1 = Avx.SetVector256(pm3[col2], pm2[col2], pm1[col2], pm0[col2],
-                                                                pm3[col1], pm2[col1], pm1[col1], pm0[col1]);
+                                                                   pm3[col1], pm2[col1], pm1[col1], pm0[col1]);
                         Vector256<float> x2 = Avx.SetAllVector256(pSrcCurrent[col1]);
                         x2 = Avx.Multiply(x2, x1);
```

Comment:
nit: Now that https://github.com/dotnet/machinelearning/pull/1292 has been merged, it would be nice to update the `Multiply, followed by Add` cases to call the `MultiplyAdd` helper.